modules/touch/src/touchinteraction.cpp

```diff
@@ -147,13 +147,20 @@ namespace {
     };
 
     constexpr openspace::properties::Property::PropertyInfo
-    ZoomBoundarySphereMultiplierInfo = {
-        "ZoomBoundarySphereMultiplier",
-        "Multiplies a node's boundary sphere by this in order to limit zoom & prevent "
+    ZoomInBoundarySphereMultiplierInfo = {
+        "ZoomInBoundarySphereMultiplier",
+        "Multiplies a node's boundary sphere by this in order to limit zoom in & prevent "
         "surface collision",
         "" // @TODO Missing documentation
     };
 
+    constexpr openspace::properties::Property::PropertyInfo
+    ZoomOutBoundarySphereMultiplierInfo = {
```

Comment:
Same here and `ZoomOutBoundarySphereMultiplierInfo` would be indented by one level